README.md

```diff
@@ -22,6 +22,7 @@ These are the middlewares included in this crate:
 - `Decompression`: Decompress response bodies.
 - `PropagateHeader`: Propagate a header from the request to the response.
 - `SensitiveHeader`: Marks a given header as [sensitive] so it wont show up in logs.
```

Comment:
just noticed this --- is this a duplicate of the entry for `SetSensitiveHeader`? we can fix that separately